frontend/app/components/mapbox/filter-features.js

```diff
@@ -0,0 +1,31 @@
+import Component from '@ember/component';
+import { computed } from '@ember-decorators/object';
+
+export default class MapboxFilterFeaturesComponent extends Component {
+  map = {}
+  layerId = null;
+  filterById = "id";
+  // map will be restricted to this array of features
+  featureIds = null;
+
+  @computed('featureIds', 'filterById')
+  get filterToFeatures(){
+    const { instance } = this.map;
+    if(this.featureIds && this.featureIds.length > 0 && (this.featureIds[0] != null)){
+      // Run through the selected featureIds and set a filter
+      // to match features with matching ids
```

Comment:
might be useful to make this comment a little more explicit about what the filter looks like, since that's not obvious unless you're a mapbox PROFESSIONAL:
```
// Compose  a MapboxGL filter like: ['in', 'id', '1', '2', '3'] using filterById and featureIds
```

Reply:
Yes, okay! 🙌